### PR TITLE
CON-2770 Grouped apprenticeship update banners

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ActionRequiredBanner.cshtml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+@if (actionRequiredBanners > ActionRequiredBanner.None)
+{
+    <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                Action required
+            </h2>
+        </div>
+        <div class="govuk-notification-banner__content">
+            @if (actionRequiredBanners.HasFlag(ActionRequiredBanner.PendingChangeForApproval))
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ActionRequiredBanner.cshtml:20]: HasFlagCheck could go inside `<p>` tag

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ChangesToThisApprenticeshipBanner.cshtml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+@{
+    var changeToApprenticeshipBanners = Model.GetChangeToApprenticeshipBanners();
+}
+
+@if (changeToApprenticeshipBanners > ChangeToApprenticeshipBanner.None)
+{
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-4">Changes to this apprenticeship</h2>
+
+    @if (changeToApprenticeshipBanners.HasFlag(ChangeToApprenticeshipBanner.PendingChangeWaitingForApproval))
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ChangesToThisApprenticeshipBanner.cshtml:15]: `HasFlag()` check could move 1 level down, inside the `<div>`

**Review by cofaulco — COMMENTED** [review]
minor comments

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ActionRequiredBanner.cshtml (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ActionRequiredBanner.cshtml:20]: That would produce an empty <p> tag when the check was not met

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ChangesToThisApprenticeshipBanner.cshtml (hunk 2 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ChangesToThisApprenticeshipBanner.cshtml:15]: That would produce an empty 'div' tag when the check was not met

**Review by cofaulco — APPROVED** [review]

**Pull request merged**